resources/metadata/cnbBuild.yaml

```diff
@@ -1,7 +1,9 @@
 metadata:
   name: cnbBuild
-  description: Executes a Cloud Native Buildpacks build for creating a Docker container.
-  longDescription: Executes a Cloud Native Buildpacks build for creating a Docker container.
+  description: Executes Cloud Native Buildpacks.
+  longDescription: |-
+    Executes a Cloud Native Buildpacks build for creating Docker container/s.
+    Please note, that the cnbBuild step gernerally works despite being in beta state, but there might be changes before we remove the beta notice.
```

Comment:
```suggestion
    **Important:** Please note, that the cnbBuild step gernerally works despite being in **beta** state, but there might be changes before we remove the beta notice.
```

Reply:
I could bet I did this comment already... But there seems to be nothing...

Reply:
`gernerally` -> `generally`

> gernerally works despite being in **beta** state

Should we really highlight that it works generally? Is that not the very definition of being `beta`?

Reply:
Maybe you are right, @c0d1ngm0nk3y. Do you have a wording in mind?